js/templates/settings.html

```diff
@@ -1261,6 +1261,35 @@ <h3 class="padding15 margin0 fontWeight500"><%= polyglot.t('Advanced') %></h3>
               </div>
 
             </div>
+            <div class="flexRow">
+
+              <div class="flexCol-3  borderRight custCol-border">
+                <div class="fieldItem">
+                  <label>
+                     Connected Peers
+                  </label>
+                </div>
+              </div>
+
+              <div class="flexCol-9 borderRight0 custCol-border">
+                <div class="fieldItem flexRow">
+                  <div class="flexRow">
+                    <div class="flexCol-9">
```

Comment:
You don't need the flexCol-9 class here. You can just have a div with the text, and the div with the button inside the flexRow.